projects/components/src/table/data/table-data-source.ts

```diff
@@ -1,8 +1,12 @@
 import { Observable } from 'rxjs';
 import { TableColumnConfig, TableFilter, TableSortDirection } from '../table-api';
 
-export interface TableDataSource<TResult, TCol extends TableColumnConfig = TableColumnConfig> {
-  getData(request: TableDataRequest<TCol>): Observable<TableDataResponse<TResult>>;
+export interface TableDataSource<TResult, TCol extends TableColumnConfig = TableColumnConfig, TDataTriggers = unknown> {
+  secondaryDataTriggers$?: Observable<TDataTriggers>; // TBD
```

Comment:
IMO, this is fine how it is used here. Adding it to watched observables in the data source is sound logic. However, does it give us the intended effect. Do we show a loader while the new data is being resolved?

Reply:
Yes.